R/sensitivity-calculation.R

```diff
@@ -269,7 +268,7 @@ sensitivityCalculation <- function(simulation,
     } else {
       # Convert tidy data to wide format
       pkParameterNames <- c(
-        names(ospsuite::StandardPKParameter),
+        names(ospsuite::allPKParameterNames()),
```

Comment:
`allPKParameterNames()` the names of all PK parameters defined in the system (incl. the user-defined PK parameters).